client/src/leap/soledad/client/encdecpool.py

```diff
@@ -253,9 +193,9 @@ def _encrypt_doc(self, doc):
         secret = self._crypto.secret
         args = doc.doc_id, doc.rev, docstr, key, secret
```

Comment:
Yes, but in this specific case I couldn't find a meaningful name for this sequence of parameters.
Without the name it's better to be explicit for now. At least it's just 5 yet.